### ISSUE TITLE
Add offline mode

### ISSUE THREAD

**Issue by johnSamilin** [johnSamilin/rosplata#64]
All actions should be stored in local database and be syncronized when there is stable connection

**Comment by johnSamilin** [comment]
service worker + request persistent storage + storage quota esitmation + various notifications about persistent storage permission

**Comment by johnSamilin** [comment]
- [ ] re-cache assets when user changes language
- [ ] disable language selector in offline mode

**Comment by okainov** [comment]
I read in readme:

>Imagine a situation: you and your friends go to a bar (possibly abroad with no internet available). You order something, have fun. In the end you get the check and no one remembers what did they order and who spent how much. This is exactly the problem Rosplata is here to solve.

And then I see this issue, so it basically means that the service is not usable offline, correct?

**Comment by johnSamilin** [comment]
> And then I see this issue, so it basically means that the service is not usable offline, correct?

Yep, right now offline mode is not ready. This issue is more like an epic (in Jira terms) for implementing full support.
